### PR TITLE
Few updates

### PR DESCRIPTION
**Pull request by BLUE-DEVIL1134**
- Changed the start time for the first video to zero
- Let people know to choose `Javascript` and not `Typescript`.
- Fixed a type on line `527`
- Updated deploy image on line `549`

### CODE REVIEW

File: Whitelist-dApp.md [hunk 1]
```diff
@@ -524,7 +524,7 @@ export const WHITELIST_CONTRACT_ADDRESS = "YOUR_WHITELIST_CONTRACT_ADDRESS";
 export const abi = YOUR_ABI;
 ```
 
-Replace `"YOUR_WHITELIST_CONTRACT_ADDRESS"` with the address of the whitelist contract that you deployed. Replace `"YOUR_ABI"` with the ABI of your Whitelist Contract. To get the ABI for your contract, go to your `hardhat-tutorial/artifacts/contracts/Whitelist.sol` folder and from your `Whitelist.json` file get the array marked under the `"abi"` key (it will be. a huge array, close to 100 lines if not more).
+Replace `"YOUR_WHITELIST_CONTRACT_ADDRESS"` with the address of the whitelist contract that you deployed. Replace `"YOUR_ABI"` with the ABI of your Whitelist Contract. To get the ABI for your contract, go to your `hardhat-tutorial/artifacts/contracts/Whitelist.sol` folder and from your `Whitelist.json` file get the array marked under the `"abi"` key (it will be. a huge array, close to 100 lines not more).
```
Comment (yanukadeneth99) [inline Whitelist-dApp.md:527]: `If not more`, is actually the right meaning, but you can change that sentence up. 

Maybe say : 
It is a huge array consisting of 100 lines or more.

File: Whitelist-dApp.md [hunk 2]
```diff
@@ -546,7 +546,7 @@ We will now deploy your dApp, so that everyone can see your website and you can
 
 - Go to [Vercel](https://vercel.com/) and sign in with your GitHub
 - Then click on `New Project` button and then select your Whitelist dApp repo
-- ![](https://i.imgur.com/ZRjfkCE.png)
+![](https://i.imgur.com/bwwmDdH.png)
```
Comment (yanukadeneth99) [inline Whitelist-dApp.md:549]: Nice catch!

**Review by yanukadeneth99 — CHANGES_REQUESTED** [review]

File: Whitelist-dApp.md (hunk 2 above)
Comment (BLUE-DEVIL1134) [reply Whitelist-dApp.md:549]: thanks!

**Comment by yanukadeneth99** [comment]
All seems to be in order @BLUE-DEVIL1134. Nice work.

**Pull request merged**